src/basic/env-util.c

```diff
@@ -776,6 +776,18 @@ int getenv_bool_secure(const char *p) {
         return parse_boolean(e);
 }
 
+int getenv_uint64_secure(const char *p, unsigned base, uint64_t *ret) {
+        const char *e;
+
+        assert(p);
+
+        e = secure_getenv(p);
+        if (!e)
+                return -ENXIO;
+
+        return safe_atollu_full(e, base, (unsigned long long *) ret);
```

Comment:
actually, just use safe_atou64() here, and drop the `base` argument. It's highly unlikely we'll use anything but decimal here.